src/@next/components/organisms/ProductVariantPicker/ProductVariantAttributeSelect.tsx

```diff
@@ -0,0 +1,149 @@
+import React from "react";
+
+import { Icon, Input } from "@components/atoms";
+import { InputSelect } from "@components/molecules";
+import { ProductDetails_product_variants } from "@sdk/queries/types/ProductDetails";
+import {
+  IProductVariantsAttribute,
+  IProductVariantsAttributesSelectedValues,
+} from "@types";
+
+import { useSelectableProductVariantsAttributeValues } from "../../../hooks/useSelectableProductVariantsAttributeValues";
```

Comment:
I believe we are moving all hooks that are used inside @next to @next/hooks (this way we also fix any type issues thanks to more strict TS config)